etc/docs/template/layouts/partials/header/main.html

```diff
@@ -0,0 +1,12 @@
+<header id="header-db" class="row" role="navigation">
+  <div class="header-content">
+    <!--logo start-->
+    <div class="logo pull-left">
+      <a href="{{.Site.BaseURL}}">
+        <img src="{{.Site.BaseURL}}/s/img/logo-mongodb-header.png", alt="MongoDB.org" />
```

Comment:
We should probably change this:
https://www.mongodb.com/brand-resources

Reply:
I'll address this with the other `img` related comments in the follow up PR!